nuimo/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(name='nuimo',
+      version='1.0.0',
+      description='Nuimo SDK for Python on Linux',
+      url='https://github.com/getsenic/nuimo-linux-python',
+      maintainer='Lars Blumberg',
+      maintainer_email='lars@senic.com',
```

Comment:
Please change `maintainer` to `Senic GmbH` and `maintainer_email` to `developers@senic.com`